target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -11,7 +11,7 @@ case "$FIRMWARE" in
 	case $board in
 	askey,rt4230w-rev6)
 		caldata_extract "art" 0x1000 0x3f1f
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x6) +1)
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x1006))
```

Comment:
If you don't increment anymore, you can drop the macaddr_add as well:
ath10k_patch_mac $(mtd_get_mac_binary ART 0x1006)

Same for the other one ...

Reply:
Err, btw, you now have the default position inside calibration data, so you should get correct MAC addresses even if you drop the ath10k_patch_mac lines entirely, as caldata_extract will read them from that location anyway. Please test!